### ISSUE TITLE
Web-морда

### ISSUE THREAD

**Issue by iillyyaa2033** [iillyyaa2033/nmud-content#6]
- [ ] Подсказки к полю ввода команд (сама команда и синтаксис)
- [ ] Боковые меню: справка

**Comment by klessh** [comment]
Всмысли?

**Comment by iillyyaa2033** [comment]
Да, точно сюда. Веб - это одноименная папка контента + API nmud-core. Так что все путем.
